packages/sdk/src/encryption/PublisherKeyExchange.ts

```diff
@@ -67,10 +75,13 @@ export class PublisherKeyExchange {
         })
         eventEmitter.on('messagePublished', (msg) => {
             if (msg.signatureType === SignatureType.ERC_1271) {
-                const address = msg.getPublisherId()
-                if (!this.erc1271ContractAddresses.has(address)) {
-                    logger.debug('Add ERC-1271 publisher', { address })
-                    this.erc1271ContractAddresses.add(address)
+                const signerUserId = msg.getPublisherId()
```

Comment:
This would not be the "recovered" signer but the erc-1271 contract. Maybe the naming here is a bit confusing?